src/components/BookList.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import Book from './Book';
+
+const BookList = () => (
+  <>
+    <table>
+      <Book />
+    </table>
+  </>
+);
+
+export default BookList;
```

Comment:
- [ ] This is where you should put the logic you put in your `Book` component, then pass a `book` object as a prop to each `Book` component inside of the `map` function. This way, your `Book` component stays "dumb", without any logic, only presenting the HTML of each book. Additionally, this component should be put inside of a folder called `containers`, where you will put all of the components that are indeed doing something special, with all of the logic.